### PR TITLE
Improve Gen 3 item/move suggestions

### PR DESCRIPTION
**Pull request by Gitrman19**
gen 3:
-remove mental herb and sitrus berry from popular items
-add salac berry to popular items
-make iron tail only not useless on steel types
-make hp fire/ice/electric only show up without elemental punches
-make hp grass show up with giga drain
-hide hp fighting when brick break
gen 6:
-make iron tail viable when no poison jab/iron head/gunk shot
misc:
-include checks for fire blast and grass knot in hp fire/grass

### CODE REVIEW

**Comment by Zarel** [comment]
It looks like you started work on this 19 days ago, and didn't incorporate your changes from #1544, so your changes now conflict with those changes.

These look like good changes, though! Can you merge those changes in? I can help you with that if you need help.

**Comment by Gitrman19** [comment]
> It looks like you started work on this 19 days ago, and didn't incorporate your changes from #1544, so your changes now conflict with those changes.
> 
> These look like good changes, though! Can you merge those changes in? I can help you with that if you need help.

I do need help. I thought you accepted my PR 19 days ago and it would ignore those changes. What do I do?

**Comment by Zarel** [comment]
https://psim.us/devdiscord is probably a good place to start.

**Comment by Gitrman19** [comment]
so what now? you said I needed to merge. also, I think I need to do something else to remove the previous, irrelevant and conflicting commits

**Comment by Zarel** [comment]
I looked more closely and it looks like there are other flaws here that also need to be addressed.

File: build-tools/build-indexes [hunk 1]
```diff
@@ -613,17 +613,27 @@ process.stdout.write("Building `data/teambuilder-tables.js`... ");
...
 			case 'heavydutyboots':
 			case 'expertbelt':
 				greatItems.push(id);
 				break;
+			case 'salacberry':
+				if (genNum === 3) greatItems.push(id);
+				else goodItems.push(id);
+				break;
+			case 'mentalherb':
+				if (genNum >= 5) greatItems.push(id);
```
Comment (Zarel) [inline build-tools/build-indexes:626]: You need an `else` here.

**Comment by Gitrman19** [comment]
okay, I can fix that. But I still don't know how to remove the irrelevant commits

**Pull request closed without merging**